### PR TITLE
Add Spell Effect for Acid Grip

### PR DESCRIPTION
**Pull request by CrackJackFlood**
Adds effect that gives a -10 status speed penalty it predicates off persistent acid damage

### CODE REVIEW

**Comment by CrackJackFlood** [comment]
I am confused by the broken link if anyone could offer advice here much appriecated. I checked all links are working.

File: packs/spell-effects/spell-effect-acid-grip.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "_id": "6TGcfVyzzVHEo7ke",
+    "img": "icons/magic/acid/dissolve-arm-flesh.webp",
+    "name": "Spell effect: Acid Grip",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Acid Grip]</p>\n<p><span style=\"font-family:Roboto, sans-serif\">A creature taking persistent damage from this spell takes a –10-foot status bonus to its Speeds.</span></p>"
```
Comment (simonward) [inline packs/spell-effects/spell-effect-acid-grip.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Acid Grip]</p>\n<p>A creature taking persistent damage from this spell takes a –10-foot status bonus to its Speeds.</p>"
```

File: packs/spells/acid-grip.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
...
+            "value": "<p>An ephemeral, taloned hand grips the target, burning it with magical acid. The target takes 2d8 acid damage plus @Damage[floor(@item.level/2)d6[persistent,acid]] damage depending on its Reflex save. A creature taking persistent damage from this spell takes a –10-foot status bonus to its Speeds. @UUID[Compendium.pf2e.spell-effects.Item.6TGcfVyzzVHEo7ke]{Spell effect: Acid Grip}</p><hr /><p><strong>Critical Success</strong> The creature is unaffected.</p>\n<p><strong>Success</strong> The creature takes half damage and no persistent damage, and the claw moves it up to 5 feet in a direction of your choice.</p>\n<p><strong>Failure</strong> The creature takes full damage and persistent damage, and the claw moves it up to 10 feet in a direction of your choice.</p>\n<p><strong>Critical Failure</strong> The creature takes double damage and full persistent damage, and the claw moves it up to 20 feet in a direction of your choice.</p><hr /><p><strong>Heightened (+2)</strong> The initial damage increases by 2d8, and the persistent acid damage increases by 1d6.</p>"
```
Comment (simonward) [inline packs/spells/acid-grip.json:30]: Looks like the extract messed up.

File: packs/spell-effects/spell-effect-acid-grip.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+    "_id": "6TGcfVyzzVHEo7ke",
+    "img": "icons/magic/acid/dissolve-arm-flesh.webp",
+    "name": "Spell effect: Acid Grip",
```
Comment (simonward) [inline packs/spell-effects/spell-effect-acid-grip.json:4]:
```suggestion
    "name": "Spell Effect: Acid Grip",
```

File: packs/spells/acid-grip.json (hunk 2 above)
Comment (CrackJackFlood) [reply packs/spells/acid-grip.json:30]: Where?

Could this have something to do with the order I extracted the packs?

Comment (simonward) [reply packs/spells/acid-grip.json:30]: Yeah. Typically extract the effects first, then the item. You should be able to just reextract the item.

File: packs/spell-effects/spell-effect-acid-grip.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            "remaster": true,
+            "title": "Pathfinder Player Core"
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "self:condition:persistent-damage:acid"
+                ],
+                "selector": "speed",
```
Comment (simonward) [inline packs/spell-effects/spell-effect-acid-grip.json:29]:
```suggestion
                "selector": "all-speeds",
```

**Pull request merged**